roles/manage_transit_gateway/README.md

```diff
@@ -0,0 +1,72 @@
+manage_transit_gateway
+==================
+
+A role to create/delete a Transit Gateway with VPC/VPN attachments.
+
+Requirements
+------------
+
+AWS User Account with the following permission:
+
+* ec2:CreateTransitGateway
+* ec2:DescribeTransitGateway
+* ec2:DeleteTransitGateway
+* ec2:CreateTransitGatewayVpcAttachment
+* ec2:DescribeTransitGatewayVpcAttachment
+* ec2:DeleteTransitGatewayVpcAttachment
+* ec2:CreateVpnConnection
+* ec2:DescribeVpnConnection
+* ec2:DeleteVpnConnection
+
+Role Variables
+--------------
+
+* **action**: Whether to create or delete the transit gateway. Choices: 'create', 'delete'. Default: 'create'.
+* **transit_gateway**: A dict of parameters needed to create transit gateway.
```

Comment:
You should either defined the elements of the dict or flatten this definition into several variables `transit_gateway_asn`, `transit_gateway_description`, `transit_gateway_tags`